test/utils/apis/kruise/apps/pub/inplace_update.go

```diff
@@ -0,0 +1,159 @@
+/*
+Copyright 2020 The Kruise Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pub
```

Comment:
we are very close to remove kruise completely, we could use v1alpha1 as the test for CRD

Reply:
AFAIK, we just need some CRDs to try to place in the test, we won't install kruise controller. I am not sure it worth to keep the kruise API just for test.